docs/use/backupRestore.md

```diff
@@ -0,0 +1,78 @@
+# Backup and restore databases
+
+## Why it’s important to create backups
+
+Databases (DBs) may get damaged due to a variety of reasons, including disk crashes, power outages, software errors, or even virus attacks.  
+
+To ensure that you can recover data in such events, it is critical to back up your databases. With Everest, you can do this with zero downtime and minimal performance impact.
+
+## Supported setups
+
+Currently, you can create and restore on-demand backups. Future releases will add Point-in-Time-Recovery and scheduled backups functionality.
+
+## Prerequisites
+
+### Prepare storage location
+
+Before working with backups, create a backup location as a backup destination for creating and storing your backup artifacts.  
+
+Everest supports Amazon S3-compatible backup locations, which means you can use [AWS S3](https://aws.amazon.com/s3/) or any other storage solutions that support S3 API, like [min.io](https://min.io/).
+
+!!! caution alert alert-warning "Important"
+
+     Make sure not to share the same backup storage location for multiple database clusters.
+
+To create a backup storage location:
+
+1. Go to **Settings > Backup storage** and click **Add backup storage**.  
+2. In the **Name** field, specify a location name using only lowercase alphanumeric characters or hyphens. 
+3. Enter your credentials, making sure to use regional AWS STS endpoints instead of the global endpoint: `https://s3.<region>.amazonaws.com`.
+ Using an endpoint that is geographically closer to your application reduces latency and provides better response times.
```

Comment:
As discussed offline we need to have somewhere in this section something stating that the S3 bucket must have [object lock](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) disabled.

Reply:
@recharte should we add that we user needs to have read/write/delete permissions for the bucket..

Reply:
Yes, we can mention that as well, thanks for the suggestion Slava